conf/test.config

```diff
@@ -20,10 +20,18 @@ params {
     max_time   = '6.h'
 
     // Input data
-    // TODO nf-core: Specify the paths to your test data on nf-core/test-datasets
-    // TODO nf-core: Give any required params for the test so that command line flags are not needed
-    input  = 'https://raw.githubusercontent.com/nf-core/test-datasets/viralrecon/samplesheet/samplesheet_test_illumina_amplicon.csv'
+    input           = 'https://raw.githubusercontent.com/sguizard/test-datasets/isoseq/samplesheet.csv'
+    primers         = 'https://raw.githubusercontent.com/sguizard/test-datasets/isoseq/testdata/primers.fasta'
+    fasta           = 'https://raw.githubusercontent.com/sguizard/test-datasets/isoseq/reference/Homo_sapiens.GRCh38.dna.chromosome.19.fasta'
+    gtf             = 'https://raw.githubusercontent.com/sguizard/test-datasets/isoseq/reference/Homo_sapiens.GRCh38.104.chr.13_18_19.gtf'
```

Comment:
Add these files to `https://github.com/nf-core/test-datasets/` in a dedicated branch, i.e. `isoseq`, if not yet existing. Ping me if you need help 😄